lib/farmoney/version.rb

```diff
@@ -1,3 +1,3 @@
 module Farmoney
-  VERSION = "0.1.1"
```

Comment:
I think this new version is wrong. It should be 0.2.0 :P

Reply:
lol what are you doing here?! No added functionality this is just a patch
https://semver.org/